### PR TITLE
replaced auto-generated setup.py with hard-coded values

### PR DESCRIPTION
**Pull request by jacobrobertbaca**
# Description of change
I'm replacing the auto-generated setup.py file with a hardcoded file. I'm doing this because pypi doesn't seem to read the autogenerated file

# Manual QA steps
 - 
 
# Risks
 - none
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+              'pylint'
+          ]
+      },
+      entry_points='''
+          [console_scripts]
+          tap-criteo=tap_criteo:main
+      ''',
+      packages=find_packages(),
+      package_data={
+          'tap_looker': [
```
Comment (luandy64) [inline setup.py]: Copy pasta

File: setup.py [hunk 2]
```diff
@@ -1,5 +1,33 @@
 """Packaging boilerplate for tap-criteo."""
 import os
-import setuptools
+from setuptools import find_packages, setup
 
-setuptools.setup()
+setup(name='tap-criteo',
+      version='0.0.1',
+      description='Singer.io tap for extracting data/metadata from the Criteo Marketing API',
+      author='judah.rand@fospah.com',
```
Comment (luandy64) [inline setup.py:8]: I don't know what the correct value is here

**Review by luandy64 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (jacobrobertbaca) [reply setup.py]: good eye

**Review by luandy64 — APPROVED** [review]
Seems to work 👍

**Review by luandy64 — APPROVED** [review]

**Pull request merged**